Sources/Nimble/Utils/Async.swift

```diff
@@ -1,10 +1,13 @@
+import CoreFoundation
+import Dispatch
 import Foundation
 
-#if _runtime(_ObjC)
-import Dispatch
+#if !_runtime(_ObjC)
+    private let NSEC_PER_SEC: CUnsignedLongLong = 1000000000
```

Comment:
If this is Swift 3, the new [namings](https://github.com/apple/swift-evolution/blob/master/proposals/0088-libdispatch-for-swift3.md#time) might make it look more like `DispatchTimeInterval.nanoseconds(1)`. I guess we'd need to move the `_runtime(_ObjC)` check lower to the callsite...?

Anyway, I'm not sure it's *better* to use `DispatchTimeInterval.nanoseconds(1)`, just thought I'd point out there's an alternative to `NSEC_PER_SEC` in Swift 3, so there might be a different way to do this here.